.github/workflows/packaging-test-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+name: Packaging Tests
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+  packaging_tests:
+    name: Packaging Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        packaging_docker_image:
+          - citus/packaging:oraclelinux-7-pg
+          - citus/packaging:oraclelinux-8-pg
+          - citus/packaging:centos-7-pg
+          - citus/packaging:centos-8-pg
+        POSTGRES_VERSION:
+          - 13
+          - 14
+          - 15
+        include:
+          - packaging_docker_image: citus/packaging:ubuntu-jammy-all
+          - packaging_docker_image: citus/packaging:debian-bookworm-all
+          - packaging_docker_image: citus/packaging:ubuntu-focal-all
+          - packaging_docker_image: citus/packaging:debian-bullseye-all
+          - packaging_docker_image: citus/packaging:ubuntu-bionic-all
+          - packaging_docker_image: citus/packaging:debian-buster-all
```

Comment:
What is the difference between this section and `packaging_docker_image` section, one for rpm and the other is for deb envs ?

Reply:
include part is added to the execution list directly but the for the upper part below matrix , there are two parameters which multiplied in cartesian (packaging_docker_image, POSTGRES_VERSION)
Since we need one docker build image for each of every postgres version for rpm based distros, we need such structure

Reply:
What I couldn't quite understand is that while we have references to some "rpm" distros in `packaging_docker_image:`, we do only have references to "deb" based distros in `include:`; so not able follow what is the distinction there.

Reply:
We have just one docker image for deb based versions however, we have one image just for each distro os pair. For example, debian-buster-all builds packages for all pg versions. However, centos7-pg10,centos7-pg11,centos7-pg12,centos7-pg13,centos7-pg14,centos7-pg15 are required for centos builds.

Reply:
we might want to add such a comment then: https://github.com/citusdata/citus/pull/6499#discussion_r1026360125

Reply:
Fixed as you suggested